spec/spec.md

```diff
@@ -154,16 +154,46 @@ was presented separately at IIW and almost immediately became a cornerstone of
 this effort.
 
 ## Presentation Exchange Context
+Presentation Exchange objects support a large variety of possible content and
+signature types. As this version of our specification has a limited scope, a
+number of considerations need to be made for
+[presentation definitions](https://identity.foundation/presentation-exchange/#presentation-definition)
+to provide what we need.
 
-### Selective Disclosure
+### format property
 
-To allow for selective disclosure of Verifiable Credential claims the use of a 
-JSON-LD frame object is combined with a `BbsBlsSignature2020` on the 
-credential. This signature allows for a zero knowledge proof of the original 
-signature, allowing the prover to derive a `BbsBlsSignatureProof2020` that will 
-verify the disclosed claims. 
+A `presentation definition` that complies with this specification MUST contain a
+`format` property. The value of the `format` property MUST be an object with the
+following property:
+- `ldp_vp`: This property indicates that a W3C Verifiable Presentation will be
+submitted in the form of a JSON object. The value of this property MUST be an
+object with a `proof_type` property that has a value of either
+`BbsBlsSignatureProof2020` or `BbsBlsBoundSignatureProof2020`.
```

Comment:
```suggestion
`BbsBlsSignature2020`, `JsonWebSignature2020`, or `Ed25519Signature2018`.
```

Reply:
the VP does not need to be related to the `derivedCredential` that is in the `verifiableCredential` property of the VP.